lib/engine/game/g_1848/game.rb

```diff
@@ -642,17 +651,20 @@ def adelaide
           @adelaide ||= hex_by_id('G6')
         end
 
-        def check_sydney_adelaide_connected
-          return @sydney_adelaide_connected if @sydney_adelaide_connected
+        def sydney_adelaide_connected?
+          return true if @sydney_adelaide_connected
 
           graph = Graph.new(self, home_as_token: true, no_blocking: true)
           graph.compute(sar)
```

Comment:
Can the graph be removed from here and just return @sydney_adelaide_connected?

Reply:
Basically, make a separate method called, check_for_sydney_adelaide_connection.

Reply:
it's only used once
<img width="442" alt="image" src="https://user-images.githubusercontent.com/108803147/182913034-656206f9-4344-4555-8a24-8487939bcf48.png">

Reply:
I added the first line as a defense in case anyone accidentally calls this method, when they just want to check sydney_adelaide_connected value
 i can rename to check_... if that's the convention

Reply:
Got it. Maybe rename so that its clear that the method is for making the check, not querying state.